src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -686,6 +776,68 @@ def cmac_algorithm_supported(self, algorithm):
     def create_cmac_ctx(self, algorithm):
         return _CMACContext(self, algorithm)
 
+    def create_x509_csr(self, builder, private_key, algorithm):
+        # TODO: check type of private key parameter.
```

Comment:
I wouldn't even xfail them. Let them fail CI since that will remind us that this PR isn't ready yet (and we can rebase/restart builds after the evp_pkey improvement merges)